jcl/src/openj9.gpu/share/classes/com/ibm/gpu/Maths.java

```diff
@@ -30,7 +30,15 @@
  * on any connected CUDA GPU. A successful sort operation
  * results in the array being sorted in ascending order.
  */
+/*[IF Java16]*/
+public final class Maths {
+
+	private Maths() {
+		super();
+	}
+/*[ELSE] Java16
 public class Maths {
+/*[ENDIF] Java16*/
```

Comment:
I'm iffy on duplicating the class definition.  How do you feel about something like: 

```suggestion
public 
/*[IF Java16]*/
final 
/*[ENDIF] Java16*/
class Maths {

/*[IF Java16]*/
	private Maths() {
		super();
	}
/*[ENDIF] Java16*/
```

Reply:
I tried that first but rejected it for two reasons:
* it involves repeating the condition
* I didn't like the formatted result (which ends up in src.zip):
```
public
final
class Maths {
```

Reply:
Neither is great.   I don't have a firm objection.